index.html

```diff
@@ -40,8 +41,8 @@ <h1 class="header-text">IdeaBox</h1>
       <section class="idea-cards">
         <article class="idea-card">
```

Comment:
Let's remember to remove the pre-loaded cards that are built into the page now that the functionality for adding/removing/favorite cards has been added.